code/modules/antagonists/spy/spy_bounty_handler.dm

```diff
@@ -75,7 +75,8 @@
 	refresh_bounty_list()
 
 /// Helper that returns a list of all active bounties in a single list, regardless of difficulty.
-/datum/spy_bounty_handler/proc/get_all_bounties() as /list
+/datum/spy_bounty_handler/proc/get_all_bounties()
+	RETURN_TYPE(/list)
```

Comment:
These syntaxes do the same thing but the former's not yet supported in OD and honestly it has no use here anyways

Reply:
https://github.com/OpenDreamProject/OpenDream/issues/1697